lib/cc/services/github_pull_requests.rb

```diff
@@ -72,19 +57,27 @@ def simple_failure(message)
     { ok: false, message: message }
   end
 
+  def response
+    @response || simple_failure("Nothing happened")
+  end
+
   def update_status_skipped
-    update_status("success", "Code Climate has skipped analysis of this commit.")
+    update_status(
+      "success",
+      "Code Climate has skipped analysis of this commit."
+    )
   end
 
   def update_status_success
     update_status("success", "Code Climate has analyzed this pull request.")
+    add_comment
```

Comment:
Not in love with having status mixed in with commenting, but it's a larger issue with this class in general (that it does two things). Commenting should go away soon, so I'm ok with this. 

However -- I think we swap the order of these lines for now. We're not capturing both results in the response back (just the last one), but the PR one is the more important one right now.